### PR TITLE
Turning counts handler and algorithm

### PR DESCRIPTION
**Pull request by runnersaw**
The turning counts algorithm I created follows a few steps:

1. Creates an array of all trajectories
2. Creates a matrix of size [num_objects, samples]. Samples are length 10 (I believe). They are 5 tuples of (x_vel, y_vel), normalized to 1. This is for clustering later, meaning that we can group objects that have like velocities
3. Clusters the trajectories together
4. Orders the clusters by straightness, and then selects the largest 2 non-similar directions as the "geometry of the intersection". This geometry is a 2-tuple as (right angle, down angle).
5. The initial and final heading of the car is calculated by taking the velocity at the beginning or end and comparing it to the intersection geometry to find the most similar road at the intersection. 

The algorithm assumes a 2-way, 2-street intersection where the roads don't curve. We can re-think how to re-work the algorithm for other cases later.

### CODE REVIEW

File: app/traffic_cloud_utils/plotting/visualization.py [hunk 1]
```diff
@@ -29,15 +29,12 @@
 
 
 
-def road_user_traj(fig, filename, fps, homographyFile, roadImageFile):
+def road_user_traj(filename, homographyFile, roadImageFile, save_path, objs_to_plot=None, plot_cars=False):
     """
     Plots all road-user trajectories.
     """
-
     homography = inv(loadtxt(homographyFile))
```
Comment (dcelik) [inline app/traffic_cloud_utils/plotting/visualization.py:36]: Why do you need to inv the homography file?

**Review by dcelik — APPROVED** [review]
This is so cool! Great job Sawyer!

File: app/traffic_cloud_utils/plotting/visualization.py (hunk 1 above)
Comment (runnersaw) [reply app/traffic_cloud_utils/plotting/visualization.py:36]: AFAIK, homography projects from pixels into the world space. We're using this to project back from world space into pixel space, so we need the inverse matrix.

Comment (dcelik) [reply app/traffic_cloud_utils/plotting/visualization.py:36]: okay makes sense. Thanks for explaining that!

**Pull request merged**